build.gradle

```diff
@@ -48,6 +48,13 @@ subprojects {
         useJUnitPlatform()
     }
 
+    java {
+        toolchain {
+            languageVersion = JavaLanguageVersion.of(16)
```

Comment:
Does this result in a JAR which is still supported by older Minecraft versions / Java versions colloquially run with older MC versions?

Reply:
A normal update because time is moving on and everyone hates outdated Java versions.

Reply:
Unfortunately, there are too many people who haven't gotten with the times. I'd love it if every server owner had decided to stop running Minecraft 1.8 (or Java 8, for that matter) but here we are. In general we'd rather keep things as friction-free as possible. We'll probably consider a move to Java 11 in the future but this isn't the time for that.